common.py

```diff
@@ -196,6 +205,32 @@ def is_pending_exists():
         return False
 
 
+def watchdog(youtube_id, job_path, margin, err_type):
+    """check is job timeout
+
+    check pending and queue timestamp to determine timeout
+
+    Args:
+        youtube_id: str
+        job_path: str
+        margin: int
+        err_type: ERR_CODE
+
+    Returns:
+        None
+
+
+    """
+    if os.path.exists(job_path):
+        now = datetime.date.today()  # 現在の時刻を取得
+        timestamp = datetime.date.fromtimestamp(int(os.path.getmtime(job_path)))
```

Comment:
```suggestion
        timestamp = datetime.datetime.fromtimestamp(int(os.path.getmtime(job_path)))
```
日付ではなく時刻の取得を想定されていると思われます。